changelog/dwarf-switch.dd

```diff
@@ -0,0 +1,8 @@
+Add `-gdwarf=<version>` compiler switch
+
+This switch allow you to specify the DWARF version you want to emit, from
```

Comment:
> This switch ...

```
Emit DWARF symbolic debug info. Supported versions are 3, 4, and 5.
```
In general, phrases like "allows you" and "you want" should not exist in technical writing. Removing them makes for better reading.